README.md

```diff
@@ -1,6 +1,7 @@
 <a href="https://opensource.newrelic.com/oss-category/#community-project"><picture><source media="(prefers-color-scheme: dark)" srcset="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/dark/Community_Project.png"><source media="(prefers-color-scheme: light)" srcset="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/Community_Project.png"><img alt="New Relic Open Source community project banner." src="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/Community_Project.png"></picture></a>
 
 # K8s Agents Operator
+[![Codecov](https://codecov.io/gh/newrelic/k8s-agents-operator/branch/main/graph/badge.svg)](https://codecov.io/gh/newrelic/k8s-agents-operator)
```

Comment:
Do we want to hardcode `main` as the branch? This links also seems to be missing a token query param. Finally, should this be on the same line as the title so it is consistent with our other repos?

Reply:
Makes sense, I have made the change